services/movie_session.py

```diff
@@ -0,0 +1,46 @@
+from django.db.models import QuerySet
+
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    queryset = MovieSession.objects.all()
+    if session_date is not None:
+        queryset = queryset.filter(show_time__date=session_date)
+    return queryset
+
+
+def get_movie_session_by_id(movie_session_id: int) -> QuerySet:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: str = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None
+) -> None:
+    queryset = MovieSession.objects.filter(id=session_id)
+    if show_time is not None:
+        queryset.update(show_time=show_time)
+    if movie_id is not None:
+        queryset.update(movie_id=movie_id)
+    if cinema_hall_id is not None:
+        queryset.update(cinema_hall_id=cinema_hall_id)
```

Comment:
Better to do this in two queries:
1. get movie session
2. set all values and than save it